catalog/dags/providers/provider_api_scripts/phylopic.py

```diff
@@ -35,15 +39,43 @@ def __init__(self, *args, **kwargs):
 
     def ingest_records(self):
         self._get_initial_query_params()
-        super().ingest_records()
+        try:
+            super().ingest_records()
+
+        # Catch 410 error caused by the build_param changing while ingestion is ongoing
+        except HTTPError as error:
+            if error.response.status_code == 410:
```

Comment:
Nit just in case we add more handlers to this `except` clause:
```suggestion
        except HTTPError as error:
            # Catch 410 error caused by the build_param changing while ingestion is ongoing
            if error.response.status_code == 410:
```